build.gradle.kts

```diff
@@ -121,6 +121,13 @@ tasks.rat {
     // Sample text file for Java quickstart
     "sdks/java/maven-archetypes/examples/sample.txt",
 
+    // Files from Delta Lake Standalone Reader https://github.com/delta-io/connectors/tree/master/standalone/src
+    "sdks/java/io/deltalake/src/main/java/io/**/*",
```

Comment:
This won't work. Files contributed to Beam need to comply with the license / ICLA and license header.